code/modules/uplink/uplink_items.dm

```diff
@@ -1160,13 +1160,13 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 			of humanoids. It has two settings: intensity, which controls the power of the radiation, \
 			and wavelength, which controls how long the radiation delay is."
 	item = /obj/item/healthanalyzer/rad_laser
-	cost = 3
+	cost = 4
 
 /datum/uplink_item/device_tools/assault_pod
 	name = "Assault Pod Targeting Device"
 	desc = "Use to select the landing zone of your assault pod."
 	item = /obj/item/assault_pod
-	cost = 30
+	cost = 40
```

Comment:
Y tho. Assault pods have a fairly niche use, and their optimal use already requires some TC for the tools necessary to scout out the station and locate the disk.

Reply:
Has guns all around it, and can land in sec. Grate for speed nukies.